src/components/file-upload/template.njk

```diff
@@ -1,16 +1,20 @@
 {% from "label/macro.njk" import govukLabel %}
 
-{#- Include label passing all label parameters, merging in `for` and `errorMessage` #}
+<div class="govuk-o-form-group">
```

Comment:
Should this have the error modifier support?

Reply:
Sorry I should have mentioned this in the commit message. It's a weird one as we don't want to add a red border to this component (confirmed by @dashouse) but I was trying to keep it consistent with the other form components in case we want to do it in the future. Thinking about it now, it's confusing, I'll take it out.